barcode/__init__.py

```diff
@@ -89,15 +108,15 @@ def get(
     return barcode
 
 
-def get_class(name: str):
+def get_class(name: str) -> type[Barcode]:
     return get_barcode(name)
 
 
 def generate(
     name: str,
     code: str,
-    writer: BaseWriter | None = None,
-    output: str | (os.PathLike | (BinaryIO | None)) = None,
+    writer: BaseWriter | None,
```

Comment:
This parameter was previously optional. With this change, it is no longer optional.

Some tests fail because of this. You can run them with `pytest`.

Reply:
I'm changing this back.

I'm not defending it, but for disclosure I think for some reason I wanted `output` to never not be `None`.  This forced me to remove the default value. Since positional args can't come after kwargs, this forced me to remove the default value on `writer`.